debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MysqlBinaryProtocolFieldReader.java

```diff
@@ -91,6 +93,7 @@ protected Object readTimestampField(ResultSet rs, int columnIndex, Column column
             return null; // Don't continue parsing timestamp field if it is null
         }
         else if (b.length() == 0) {
+            logInvalidValue(rs, columnIndex, b);
             LOGGER.warn("Encountered a zero length blob for column index {}", columnIndex);
```

Comment:
I wonder if it makes sense in this and other error messages to add a hint that the problematic value cn be printed at the TRACE log level.

Reply:
Sure, I assume you're referring to adding the hint to the existing `warn` message.  I'll update the PR accordingly.